docker/compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.4'
+
+services:
+  consul:
+    image: consul
+    ports:
+    - "8500:8500"
+
+  frontend:
+      build: ./project-registry-frontend/
+      ports:
+      - "4200:80"
+      environment: 
+      - REGISTRY_URL="gateway:8085/api/"
+
+  gateway:
+    build: ./project-registry-gateway/
+    ports:
+    - "8085:8085"
+    environment:
+    - SPRING_CLOUD_CONSUL_HOST=consul
+    #- SPRING_APPLICATION_JSON="{'spring':{'cloud':{'gateway':{'globalcors':{'corsConfigurations':{'\'[**]\'':{'allowedOrigins':'frontend:80'}}}}}}}"
```

Comment:
one of the multiline string syntaxes would fit better here, assuming this comment is meant as an invitation to show devs how to put something like that in.

another approach that doesn't invite them to make untracked changes to this file (which is a little messy if this file needs updates later, but arguably ok) would be with (1 or more) `env_file` pointing to a gitignored file path. then devs can put whatever they want in that env file.

Reply:
This was actually a line that we were testing to see if it fixed one of our problems and forgot to delete.

Reply:
sounds good. when you leave a reply like that that in your opinion resolves the conversion, you don't need to wait for permission to click "resolve conversation" below here.